composer.json

```diff
@@ -3,7 +3,7 @@
     "type": "phpbb-extension",
     "description": "phpBB Ideas extension",
     "homepage": "https://www.phpbb.com",
-    "version": "2.0.0-dev",
+    "version": "3.0.0-dev",
```

Comment:
If you're using semver, you're allowed to break compatibility in unstable versions as much as you like. `2.0.0-dev2` can be a complete rewrite of `2.0.0-dev`. That said, if there are a lot of users, probably not a great idea. (there aren't, though)